include/linux/kfence.h

```diff
@@ -19,6 +20,20 @@ struct kmem_cache;
 #define KFENCE_NUM_OBJ ((1 << KFENCE_NUM_OBJ_LOG) - 1)
 
 extern char __kfence_pool_start[];
+extern struct static_key_false kfence_allocation_key;
+
+// TODO(elver): Shorten function names, and make __kfence_alloc private.
+// Simply rename them to __kfence_alloc and kfence_alloc respectively.
+
+void *__kfence_alloc(struct kmem_cache *s, size_t size, gfp_t flags);
+
+// TODO(elver): Add API doc.
+static __always_inline void *kfence_sampled_alloc_with_size(struct kmem_cache *s, size_t size,
```

Comment:
This one was meant to be `kfence_alloc`, now that you already renamed to `__kfence_alloc` above.

I think what I meant by making "private" is adding the __ in front, which seems to be the kernel way of denoting internal things. Renaming this means you'll have resolved the above TODO. Thank you!

Reply:
Yes, I forgot to fix that one. Will do.